src/meta/src/barrier/mod.rs

```diff
@@ -577,9 +577,14 @@ where
             // consistency.
             // Even if there's no actor to recover, we still go through the recovery process to
             // inject the first `Initial` barrier.
```

Comment:
The change will break this and there's no Initial barrier to be sent. 🥵

Reply:
I see, then I guess simply adding a sleep in the test and left the initial recovery unchanged should also works. 🥵